app/models/consent/default.rb

```diff
@@ -0,0 +1,65 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Consent::Default
+  def initialize(client:)
+    @client = client
+  end
+
+  def no_release_string
+    'None on file'
+  end
+
+  def revoked_consent_string
+    ''
+  end
+
+  def full_consent_string
+    ''
+  end
+
+  def partial_release_string
+    'Limited CAS Release'
+  end
+
+  def full_release_string
+    'Full HAN Release'
+  end
+
+  def release_current_status
+    consent_text = if @client.housing_release_status.blank?
+      @client.class.no_release_string
```

Comment:
Should this use the local `no_release_string` method?

Reply:
yes, will add this in the next commit